src/Task/YamlLint.php

```diff
@@ -56,12 +59,20 @@ public function canRunInContext(ContextInterface $context)
      */
     public function run(ContextInterface $context)
     {
-        $files = $context->getFiles()->name('/\.(yaml|yml)$/i');
+        /** @var array $config */
+        $config = $this->getConfiguration();
+        $whitelistPatterns = isset($config['whitelist_patterns']) ? $config['whitelist_patterns'] : [];
+        $extensionPattern = '/\.(yaml|yml)$/i';
```

Comment:
In most tasks, this variable is named `$extensions` (for consistency)

Reply:
The variable name has been changed : $extensionPattern -> $extensions